### ISSUE TITLE
Caret Colors all Black and can't be changed

### ISSUE THREAD

**Issue by keithn** [jaredpar/VsVim#2351]
For some reason my Vsvim caret colors are all black ( on near black theme, so makes it very hard to see the cursor) and in the options, I can select different colors, however once I go Ok, it remains black, going back into the options, they have all reverted to black

**Comment by keithn** [comment]
![image](https://user-images.githubusercontent.com/86080/44634488-ba6fcd80-a9ee-11e8-929d-44276d85885b.png)

attached is pic of the settings....

if I change the color 
![image](https://user-images.githubusercontent.com/86080/44634502-dd9a7d00-a9ee-11e8-88b1-79bbc5d96674.png)

then it resets back to the top one after ok

**Comment by ricksladkey** [comment]
It looks like you are using the VsVim from the [Open Vsix Gallery](http://vsixgallery.com/extension/VsVim.Microsoft.e214908b-0458-4ae2-a583-4310f29687c3/) which corresponds to the head of the master branch. That explains why you have block caret foreground/background colors, which is not a feature of VsVim 2.5.0. I'm not getting these symptoms, but since I implemented the caret colors feature recently, I am concerned about the behavior you are seeing.

As an experiment, can you try setting the colors in "Tools -> Options -> Environment -> Fonts and Colors":

![image](https://user-images.githubusercontent.com/2680524/44635582-85bf3000-a974-11e8-8932-d60a35774b21.png)

**Comment by keithn** [comment]
was just coming back to say, I was mucking around trying to get it to work, just as desperation move, I uninstalled easymotion ( since I had it disabled anyways ) and after that when I went back in, all my colors were correct.   I had restarted VS a number of times before uninstalling EasyMotion.    I'm pretty sure I didn't do anything else, but not sure if it was directly easy motion, or something about running VSIX installer/uninstaller  or just a coincidence.   I'll close the issue as I'm guessing it's something unique to my setup.
